### PR TITLE
Move RHV4 product to be el8 based

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update content Hypervisor CPE product
- Update profile with RHEL8 specific rules and their applicabilities
- [x] Document that `rhv4` product applies to el8 based hosts
- ~~[ ] Update prodtype in other rules.~~

#### Rationale:

- Content for el7 based hosts is provided in `rhel7` product
- Content for el8 based hosts is provided by `rhv4` product

Related to: https://lists.fedoraproject.org/archives/list/scap-security-guide@lists.fedorahosted.org/thread/CSNMJJWYPAUVBCQFCRIJHJ7PHOBE6SDB/

#### References
- https://github.com/ComplianceAsCode/content/pull/4779
- https://github.com/ComplianceAsCode/content/pull/3742
- https://github.com/ComplianceAsCode/content/pull/5024
- https://github.com/ComplianceAsCode/content/pull/3677
- https://github.com/ComplianceAsCode/content/pull/5124

### CODE REVIEW

**Comment by yuumasato** [comment]
I'd appreciate feedback on contents of `rhv4/README.md`. Do you think something else should be there?

I have realized now that the `rhel7/profiles/rhelh-stig.profile` and `rhv4/profiles/rhvh-stig.profile` are very identical, and that there exists a `rhel8/profiles/rhelh-stig.profile`.

Maybe `rhv4/profiles/rhvh-stig.profile` needs to be updated to be based on `rhel8` profile, and the same applies to `rhvh-vpp.profile`.  Thoughts on this?

**Comment by yuumasato** [comment]
For easy of review, I've dropped ee5ed7d, and will propose it separately.

The commit just did bulk labeling, any rule that applied to `rhel8` was made applicable `rhv4`.
And any rule that was only applicable to `rhel7` was made sure not to apply to `rhv4`

File: rhv4/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Red Hat Virtualization 4
+
+As oVirt moves to be based on el8, the contents of `rhv4` will also move to be based on el8.
+
+If you need content for oVirt based on el7, use the Red Hat Enterprise Linux 7 (rhel7) content.
```
Comment (jan-cerny) [inline rhv4/README.md]: My understanding was that oVirt is the open-source project and Red Hat Virtualization 4 is based on top of that. This content is for Red Hat Virtualization. oVirt isn't mentioned anywhere, so I would prefer to not mention it in the README.

Comment (yuumasato) [reply rhv4/README.md]: Thanks for your feedback, updated.

**Comment by yuumasato** [comment]
So, I think it makes sense to update the profile to be based on RHEL8 versions.
So in 3845826 I update VPP profile to be aligned with RHEL8 OSPP profile.
And in 5b2d981 I update STIG profile to be aligned with RHEL8 STIG profile.

Unfortunately this makes 0665267 mandatory, as it now enables a few rules needed by the profile.
Reviewing commit by commit will be easier than checking whole list of changed files.

**Comment by yuumasato** [comment]
> So in [3845826](https://github.com/ComplianceAsCode/content/commit/3845826716d1b5050c23a9ff3c70bdc7ee149c77) I update VPP profile to be aligned with RHEL8 OSPP profile.
> And in [5b2d981](https://github.com/ComplianceAsCode/content/commit/5b2d9819003cb6381ec539476c7446c997201e6a) I update STIG profile to be aligned with RHEL8 STIG profile.

@redhatrises, @shawndwells Would appreciate your feedback here

**Comment by redhatrises** [comment]
A couple of thoughts:

1. I noticed that any rule for RHEL8 is selected and enabled for rhv. Let's not do this as we are currently in the process with the RHV team to go through the NIST controls to evaluate applicability to RHV. Once controls are identified, we then add and enable them for RHV.

2. Let's not make the OSPP and STIG rules the same. Keep them as is for now. We can open a draft PR for these changes separately. Reason being is that RHV is going through the NIAP process now and OSPP and VPP and STIG are not the same. And since we are evaluating all of NIST 800-53 and VPP, I am hestitant to make these type of changes to the profiles right now. For example, Gnome shouldn't be installed or configured at all on RHVH or RHVM.

**Comment by redhatrises** [comment]
Also don't create a separate README integrate the comments into the main README or in the docs.

**Comment by yuumasato** [comment]
Ok, dropped commits that migrate profiles and all rules to el8.

> Also don't create a separate README integrate the comments into the main README or in the docs.

I don't know where would be a good place for such information to be. I moved it to be in the User Guide.
@redhatrises  If that is not a good place, suggest one, please.

**Comment by yuumasato** [comment]
Conflicts resolved.

**Comment by JAORMX** [comment]
/test all

ocp4 e2e test flake

**Comment by yuumasato** [comment]
I have no idea how to interpret error in `ci/prow/e2e-aws-moderate`.
`release "release-latest" failed: could not create watcher for pod: Get https://172.30.0.1:443/api/v1/namespaces/ci-op-tgyw9i7j/pods?fieldSelector=metadata.name%3Drelease-latest&watch=true: unexpected EOF`

Is it a deal breaker?

**Comment by matejak** [comment]
The PR has been approved, and those failing tests don't indicate that there is something wrong with it, so I am merging it.

**Pull request merged**